server/node_filter.go

```diff
@@ -59,27 +59,32 @@ func (nf *NodeFilter) InCluster(address string) bool {
 	return ok
 }
 
-// ControlPlane returns control plane nodes.
-func (nf *NodeFilter) ControlPlane() []*cke.Node {
+// AllNode returns nodes.
+func (nf *NodeFilter) AllNodes() []*cke.Node {
+	return nf.cluster.Nodes
+}
+
+// ControlPlanes returns control plane nodes.
+func (nf *NodeFilter) ControlPlanes() []*cke.Node {
```

Comment:
It would be appropriate to use the singular form.

https://kubernetes.io/docs/concepts/overview/components/
> A Kubernetes cluster consists of a control plane and one or more worker nodes.